src/main/java/edu/gonzaga/EpidemicCard.java

```diff
@@ -2,7 +2,56 @@
 
 public class EpidemicCard extends Card {
     
-    public void triggerEpidemic(){
+    /**
+     * Constructor for the EpidemicCard class
+     * @return an EpidemicCard object
+     * 
+     * @Author Tony
+     */
+    public void triggerEmpidemic(Gameboard board) {
+        // increase the infection rate
+        increaseInfectionRate(board);
 
+        // infect
+        infect(board);
+
+        // intensify
+        intensify(board);
+    }
+
+    /**
+     * Infects cities
+     * 
+     * @param board the gameboard
+     * @return void
+     * @Author Tony
+     */
+    private void infect(Gameboard board) {
```

Comment:
After the epidemic card, it shouldn't immediately start the infection turn, that'll already happen so this doesn't need to be here